libs/core/src/components/sage-avatar/sage-avatar.tsx

```diff
@@ -0,0 +1,124 @@
+import { Component, Host, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'sage-avatar',
+  styleUrl: 'sage-avatar.scss',
+  shadow: true,
+})
+export class SageAvatar {
+
+  /**
+   * The alt for a custom user image.
+   * @defaultValue null
+   */
+  @Prop() alt?: string | null;
+
+  /**
+   * Determines whether the badge is visible or not.
+   * @defaultValue false
+   */
+  @Prop() badge? = false;
+
+  /**
+   * Determines whether the avatar functions as a dropdown trigger.
+   * @defaultValue false
+   */
+  @Prop() dropdown? = false;
+
+  /**
+   * The src for a custom user image.
+   * @defaultValue null
+   */
+  @Prop() image?: string | null = null;
+
+  /**
+   * Size of the avatar. Value can be preset or custom.
+   */
+  @Prop({ reflect: true }) size?: 
```

Comment:
Should we document the preset sizes in some way? The default displays in the props table but the others are not.